bucket/azuredatastudio-insiders.json

```
@@ -1,22 +1,20 @@
 {
     "homepage": "https://docs.microsoft.com/en-us/sql/azure-data-studio",
-    "version": "1.12.2",
+    "version": "nightly",
     "description": "A cross-platform database tool for data professionals using the Microsoft family of on-premises and cloud data platforms.",
     "license": "MIT",
-    "bin": "bin\\azuredatastudio.cmd",
+    "bin": [
```

Why did you change this?

```suggestion
    "bin": "bin\\azuredatastudio.cmd",
```

---

Then just change the file name and do not add additional rename.

Edited original suggstion